extensions/blocks/conversation/components/context.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
+
+export default createContext();
```

Comment:
Missing new line at end of file

Reply:
fixed here https://github.com/Automattic/jetpack/pull/17916/commits/28abcf28cbc2d6455e5043732b745db0d7f0d9b5